package.json

```diff
@@ -36,7 +36,8 @@
     "test": "npx jest",
     "test-storybook": "test-storybook",
     "chromatic": "chromatic",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "test:types": "tsc -p tsconfig.json"
```

Comment:
Do we want to add this to our `pre-commit` hooks and Github-Actions?

Reply:
Yes, we should! I'll include it here. 🎉